swig/java/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+set(JAVA_SWIG_BINDING libyangJava)
```

Comment:
Please rename the library to "libyang", for that you will also need to remove the line:
`set_target_properties(${SWIG_MODULE_${JAVA_SWIG_BINDING}_REAL_NAME} PROPERTIES PREFIX "")`,
rename the interface `libyangJava.i` to `libyang.i` and the load library name in examples and tests.

Reply:
but if I modify the first line like this:
`set(JAVA_SWIG_BINDING libyang)`
and remove the line:
`set_target_properties(${SWIG_MODULE_${JAVA_SWIG_BINDING}_REAL_NAME} PROPERTIES PREFIX "")`
and rename the interface `libyangJava.i` to `libyang.i`
the target java so file will be named `liblibyang.so`, it sounds strange 

if I don't remove the line:
`set_target_properties(${SWIG_MODULE_${JAVA_SWIG_BINDING}_REAL_NAME} PROPERTIES PREFIX "")`
the target java so file will be named `libyang.so`, it will conflict with the libyang.so in the src/build directory.

Reply:
You are right, I forgot to run "make install". Then can you change it so that `System.loadLibrary` uses libyangJava name?

Reply:
The name of the generated .so file now is libyangJava.so, the code for invoking the dynamic link library libxxx.so in Java is as follows:
`System.loadLibrary(xxx);`

Reply:
OK.